tests/Earthfile

```diff
@@ -1474,15 +1474,20 @@ test-shared-cache:
       --extra_args=" --secret=content=filecontents " \
       --output_contains="filecontents"
 
-test-with-docker-parallel:
-  LOCALLY
-  RUN cd with-docker-parallel; \
-    f=/tmp/earthly.output-$(cat /proc/sys/kernel/random/uuid); \
-    earthly -P +parallel 2>&1 | tee -a $f; \
-    order=$(cat $f | grep -Eo "### [[:digit:]]+"  | cut -d' ' -f2- | tr '\n' ' ' | tr -d '[:space:]'); \
-    rm $f; \
-    echo order:$order; \
-    (echo $order | grep -Eq ^1+2+3+4+5+$) && echo "no parallelization" && exit 1 || echo "parallelization";
+test-new-visitor:
+  RUN apk update; apk add --upgrade sed
```

Comment:
is it possible to put this in the base instead: https://github.com/earthly/earthly/blob/2a9f2b10beec627d7a43238a9adb6516a833416d/Earthfile#L469